bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -4,6 +4,8 @@
 
 {% add_lang_files "firefox/new/quantum" %}
 
+{% from "macros.html" import fxa_link_fragment, fxa_email_form with context %}
```

Comment:
Nit: should this come before `add_lang_files`? I'm never quite clear if there's a good reason to do this, but it seems to be what most templates follow?